src/main/java/io/pakland/mdas/githubstats/infrastructure/shell/components/OrganizationComponent.java

```diff
@@ -0,0 +1,22 @@
+package io.pakland.mdas.githubstats.infrastructure.shell.components;
+
+import io.pakland.mdas.githubstats.domain.service.GetOrganizationFromId;
+import org.springframework.shell.standard.ShellComponent;
+import org.springframework.shell.standard.ShellMethod;
+import org.springframework.shell.standard.ShellOption;
+
+@ShellComponent
+public class OrganizationComponent {
+
+    final private GetOrganizationFromId getOrganizationFromId;
+
+    public OrganizationComponent(GetOrganizationFromId getOrganizationFromId) {
+        this.getOrganizationFromId = getOrganizationFromId;
+    }
+
+    @ShellMethod(key = "getOrg", value = "get organization info")
+    public void getOrg(@ShellOption(value = "--id") Long organizationId) {
+        getOrganizationFromId.execute(organizationId);
+    }
+
+}
```

Comment:
I do not see the need of this.

It is not possible to do this. When the user wants to search for a team, the use case should run and the flow would start. It is not possible to get an organization given an id.

Reply:
Removed ✅

Reply:
Agreed

Reply:
Ok, this is just a intermediate use case, that it isn't exposed as a domain use case, like the teacher told us in last class. Yesterday I didn't think about it.